src/pages/instances/InstanceOverview.tsx

```diff
@@ -84,19 +94,25 @@ const InstanceOverview: FC<Props> = ({ instance }) => {
               <tr>
                 <th className="u-text--muted">Cluster member</th>
                 <td>
-                  {settings?.environment?.server_clustered
-                    ? instance.location
-                    : "-"}
+                  {settings?.environment?.server_clustered &&
+                  instance.location ? (
+                    <ResourceLabel
+                      type="cluster-member"
+                      value={instance.location}
+                    />
+                  ) : (
+                    "-"
+                  )}
                 </td>
               </tr>
               <tr>
                 <th className="u-text--muted">Root storage pool</th>
                 <td>
-                  <Link
+                  <ResourceLink
+                    type="pool"
+                    value={rootPool}
```

Comment:
I think we should include the pool under devices below. Maybe mark it as type "disk (root)" or alike. But it seems wrong to have this separated out in the general section. Because in configuration, we treat the root disk like the other devices, and conceptually it is a device. So let's move it down?

Reply:
Yeah sounds good to me :+1: